configuration-wire/precomputed-v1.json

```diff
@@ -0,0 +1,73 @@
+{
+    "version": 1,
+    "precomputed": {
+        "subjectKey": "test-subject-key",
+        "subjectAttributes": {
+            "device": "iPhone",
+            "country": "USA"
+        },
+        "fetchedAt": "2024-11-18T14:23:39.456Z",
+        "response": {
```

Comment:
`response` should be an encoded string right?

Reply:
The intention was for tests to read the obfuscated json file and use this json file to check that decoding and deobfuscation worked correctly in the SDK code

So yeah this is a string in precomputed-v1-obfuscated.json and in this file it is parsed and without md5 hashing and without base64 encoding

Reply:
Could we rename this file to highlight that it's a "deobfuscated" config and not a valid configuration wire? Maybe `precomputed-v1-deobfuscated.json`?

Reply:
Sure, I can do that!

Reply:
Renamed!

`precomputed-v1.json` -> `precomputed-v1-deobfuscated.json`
`precomputed-v1-obfuscated.json` -> `precomputed-v1.json`

So now the obfuscated version will represent the official wire format, and deobfuscated is the special representation for testing purposes

Reply:
cool. I get it now. Thanks!